packages/server/src/user/roles.guard.ts

```diff
@@ -0,0 +1,31 @@
+import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common';
```

Comment:
this should probably be in the auth folder

Reply:
fixed